### PR TITLE
feature: add a test case on Kind

### PR DESCRIPTION
**Pull request by shaowenchen**
Signed-off-by: shaowenchen <mail@chenshaowen.com>

### CODE REVIEW

File: .github/workflows/kind-test.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - uses: actions/checkout@v2
+      - name: Build KubeEye
+        run: make ke
+      - name: Creating kind cluster
+        uses: helm/kind-action@v1.0.0-rc.1
+      - name: Run KubeEye
+        run: |
+          rm -rf temp & mkdir temp
+          mv ke temp/
+          ./temp/ke diag
```
Comment (LinuxSuRen) [inline .github/workflows/kind-test.yaml:24]: Why don't use `ke` directly? Make them to one line: `./ke diag`.

And maybe we need to check the exit code here. Make sure the action is failed if the exit code is not 0.

Comment (shaowenchen) [reply .github/workflows/kind-test.yaml:24]: This is to test packr2.

Comment (LinuxSuRen) [reply .github/workflows/kind-test.yaml:24]: > This is to test packr2.

I dont' understand this. No matter what, remove the temp dir is not related with `packr2`. If you try to clean the dir, maybe try to use `make clean ke`.

Comment (shaowenchen) [reply .github/workflows/kind-test.yaml:24]: Packr is a simple solution for bundling static assets inside of Go binaries. It needs to change the directory to test it works.

**Pull request merged**